browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1482,6 +1482,23 @@ L.CanvasTileLayer = L.Layer.extend({
 				var tooltipInfo = JSON.parse(textMsg.substring(textMsg.indexOf('{')));
 				this._map.uiManager.showDocumentTooltip(tooltipInfo);
 			}
+			else if (tooltipInfo.type === 'autofillpreviewtooltip') {
+
+				var strTwips = textMsg.match(/\d+/g);
+				if (strTwips != null && this._map.isEditMode()) {
+					var topLeftTwips = new L.Point(parseInt(strTwips[0]), parseInt(strTwips[1]));
+					var offset = new L.Point(parseInt(strTwips[2]), parseInt(strTwips[3]));
+
+					var topLeftPixels = this._twipsToCorePixels(topLeftTwips);
+					var offsetPixels = this._twipsToCorePixels(offset);
+					this._cellAutoFillAreaPixels = L.LOUtil.createRectangle(topLeftPixels.x, topLeftPixels.y, offsetPixels.x, offsetPixels.y);
```

Comment:
can we somehow put this into event's data? in the `map.fire ( ... )`
so we will not need to store it inside canvas layer class

Reply:
I think this is done(?).